sdl_viewer/src/lib.rs

```diff
@@ -451,7 +451,7 @@ pub fn run<T: Extension>(octree_factory: OctreeFactory) {
     // We need to create a context now, only after can we actually legally load the gl functions
     // and query 'gl_attr'.
     let _context = window.gl_create_context().unwrap();
-    video_subsystem.gl_set_swap_interval(1);
+    let _swap_interval = video_subsystem.gl_set_swap_interval(1);
```

Comment:
nit: use `sdl2::video::SwapInterval::VSync` instead of `1`?
see: https://docs.rs/sdl2/0.32.2/src/sdl2/video.rs.html#542

Reply:
Done